package.json

```diff
@@ -23,8 +23,8 @@
     "dev": "yarn run storybook",
     "build": "yarn run build:cjs && yarn run build:es && yarn run build:umd",
     "build:umd": "BABEL_ENV=umd rollup --config scripts/rollup.config.js",
-    "build:cjs": "BABEL_ENV=cjs babel src --out-dir cjs/ --ignore src/index.es.js,__tests__,__mocks__ --quiet",
-    "build:es": "BABEL_ENV=es babel src --out-dir es/ --ignore src/index.es.js,__tests__,__mocks__ --quiet && BABEL_ENV=es babel src/index.es.js --out-file es/index.js --quiet",
+    "build:cjs": "BABEL_ENV=cjs babel src --extensions '.js,.ts,.tsx' --out-dir cjs/ --ignore 'src/index.es.js','**/__tests__','**/__mocks__' --quiet",
+    "build:es": "BABEL_ENV=es babel src --extensions '.js,.ts,.tsx' --out-dir es/ --ignore 'src/index.es.js','**/__tests__','**/__mocks__' --quiet && BABEL_ENV=es babel src/index.es.js --out-file es/index.js --quiet",
```

Comment:
could this config not live in the babelrc file?

Reply:
It probably could but I figured `.babelrc` sets the environment, it shouldn't give instructions on how to bundle the package.